schema/codelists/awardStatus.csv

```diff
@@ -1,5 +1,7 @@
-Code,Title,Description
-pending,Pending,"This award has been proposed, but is not yet in force. This can be due to a cooling off period, or some other process."
-active,Active,"This award has been made, and is currently in force."
-cancelled,Cancelled,This award has been cancelled.
-unsuccessful,Unsuccessful,"This award could not be successfully made. If items or supplier details are included within the award section, then these narrow the scope of the unsuccessful award (i.e. the award of noted items, or an award to the noted supplier, was unsuccessful, but there can be other successful awards for different items listed in the tender, or to different suppliers)."
+Code,Title,Description,Deprecated,Deprecation note
+pending,Pending,"This award has been proposed, but is not yet in force. This can be due to a cooling off period, or some other process.",1.2,
+active,Active,"The bid submission deadline has passed, but the award has not yet been made or the standstill period, if any, has not yet ended.",,
+complete,Complete,"The award has been made and the standstill period, if any, has ended.",,
+cancelled,Cancelled,"This award has been cancelled by the buyer or the procuring entity (e.g. because of a change in needs, insufficient funds, or technical or procedural errors) after the bid submission deadline, but before the award had been made or before the standstill period, if any, had ended.",,
```

Comment:
```suggestion
cancelled,Cancelled,"The award was cancelled by the buyer or the procuring entity (e.g. because of a change in needs, insufficient funds, or technical or procedural errors) after the bid submission deadline, but before the award was made or before the standstill period, if any, had ended.",,
```